doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -1224,6 +1224,64 @@ Controlling the locality of commands
      occurs in a section.  The :cmd:`Set` and :cmd:`Unset` commands belong to this
      category.
 
+.. _controlling-typing-flags:
+
+Controlling Typing Flags
+----------------------------
+
+.. flag:: Guard Checking
+
+   This option can be used to enable/disable the guard checking of
+   fixpoints. Warning: this can break the consistency of the system, use at your
+   own risk. Decreasing argument can still be specified but the decrease is not
```

Comment:
The decreasing argument should be specified as it will impact the reduction rule.